recipe/meta.yaml

```diff
@@ -44,16 +44,14 @@ requirements:
     - pyarrow
     - fsspec
     - yaspin
+    - fastpdb
 
 test:
   imports:
     - polaris
   commands:
-    - pip check
```

Comment:
Exactly why I wanted you to review this 😄 

See this error - https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1075489&view=logs&j=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&t=9eb77fd2-8ddd-5444-8fc0-71cb28dcb736

I'm not convinced this has anything to do with the `polaris` conda build and so I removed it. Did we have any reason to run this before?

Reply:
I though this was one of those things we've just copied over, but turns out it is not, see e.g. [datamol's feedstock](https://github.com/conda-forge/datamol-feedstock/blob/main/recipe/meta.yaml#L47-L49).

The main error seems to be:
```
2024-11-08T18:11:22.2714825Z + pip check
2024-11-08T18:11:22.7877371Z hatch-cython 0.5.1 requires cython, which is not installed.
2024-11-08T18:11:22.7878820Z hatch-cython 0.5.1 requires hatch, which is not installed.
2024-11-08T18:11:22.7884534Z hatch-cython 0.5.1 requires hatchling, which is not installed.
2024-11-08T18:11:24.6273251Z WARNING: Tests failed for polaris-0.8.0-pyha308f57_0.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken
```

Assume this is coming from `biotite` (through `fastpdb`), see e.g. [here](https://github.com/biotite-dev/biotite/blob/main/pyproject.toml#L141-L149). 

Seems like we could solve this by including `cython`, `hatch` and `hatchling` in the build environment. I would also be okay with removing it though!

Reply:
I think that'd work!

I'm not convinced the test has any relevance though because I'd expect the conda build to fail if something were wrong (either during the build or after installed and the tests are run). Let's go without it for now.